spec/controllers/identify_controller_spec.rb

```diff
@@ -158,11 +158,11 @@
               openssl_errors: 'error 20 at 0 depth lookup: unable to get local issuer certificate',
               ficam_openssl_valid: false,
               ficam_openssl_errors: 'error 20 at 0 depth lookup: unable to get local issuer certificate',
+              matched_policy_oids: ['2.16.840.1.101.2.1.11.9'],
```

Comment:
I left a question on the ticket in response to your comment about using an array here vs. another structure like a hash and whether we can aggregate on it.